### PR TITLE
Add styling for reports

### PR DESCRIPTION
**Pull request by kehh**
Tweak some of the default styling for reports, specifically add images, tidy up css

### CODE REVIEW

File: app/printTemplates/summary/local/ca_objects_summary.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ * GNU General Public License. (http://www.gnu.org/copyleft/gpl.html). See
+ * the "license.txt" file for details, or visit the CollectiveAccess web site at
+ * http://www.CollectiveAccess.org
+ *
+ * -=-=-=-=-=- CUT HERE -=-=-=-=-=-
+ * Template configuration:
+ *
+ * @name Object summary
+ * @type page
+ * @pageSize A4
```
Comment (kehh) [inline app/printTemplates/summary/local/ca_objects_summary.php:31]: cos we like A4 not letter ;)

File: app/printTemplates/summary/local/ca_objects_summary.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<?php
+/* ----------------------------------------------------------------------
+ * app/templates/summary/local/ca_objects_summary.php
```
Comment (kehh) [inline app/printTemplates/summary/local/ca_objects_summary.php:3]: This is a copy of https://github.com/kehh/providence/blob/c7116f5a337eff3167d3340b6717571fd4ea8266/app/printTemplates/summary/ca_objects_summary.php with some modifications

File: app/printTemplates/summary/local/ca_objects_summary.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+	print $this->render("footer.php");
+?>
+	<br/>
+	<div class="title">
+		<?php print $t_item->getLabelForDisplay();?>
+	</div>
+	<div class="representationList">
+		
+<?php
+	$va_reps = $t_item->getRepresentations(array("thumbnail", "medium", "original"));
```
Comment (kehh) [inline app/printTemplates/summary/local/ca_objects_summary.php:60]: Added `original` so I can provide URLs to the images

File: app/printTemplates/summary/local/ca_objects_summary.php [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+		}
+		$va_urls[] = $va_rep['urls']['original'];
+		$vn_i ++;
+	}
+	?>
+	</div>
+<?php
+if ($va_urls) :
+  ?>
+  <div class="data"><span class="label">Image Links</span><span class='meta'>
```
Comment (kehh) [inline app/printTemplates/summary/local/ca_objects_summary.php:81]: This is extra compared to upstream

File: app/printTemplates/summary/local/footer.php [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+ */
+
+$t_item = $this->getVar('t_subject');
+
+if ($this->request->config->get('summary_header_enabled')) {
+	$vs_footer = '<table class="footerText" style="width: 100%;"><tr>';
+	if ($this->request->config->get('summary_show_identifier')) {
+		$vs_footer .= "<td class='footerText'>" . $t_item->getLabelForDisplay() . " (" . $t_item->get($t_item->getProperty('ID_NUMBERING_ID_FIELD')) . ")</td>";
+	}
+	/* @var $t_display ca_bundle_displays */
```
Comment (kehh) [inline app/printTemplates/summary/local/footer.php:42]: I added the next 4 lines compared to upstream.  This adds the name of the display to the footer. The rest of this file is copypasta, with reformatting.

File: app/printTemplates/summary/local/pdf.css [hunk 6]
```diff
@@ -1,23 +1,143 @@
-#header { position: fixed; left: 0px; top: -25px; right: 0px; height: 55px;  text-align: left; vertical-align: top; margin: 0px 5px 10px 0px; padding: 0px 5px 2px 0px; }
-#footer { position: fixed; left: 0px; bottom: -50px; right: 0px; height: 20px; text-align: center; vertical-align: bottom; margin: 0px 5px 10px 5px; padding: 3px 5px 2px 10px; }
-#header .pagingText:after { content: counter(page, decimal); }
-#body { z-index: 30000; position: relative; }
+#header {
```
Comment (kehh) [inline app/printTemplates/summary/local/pdf.css:1]: Replaced with the version from app/printTemplates/summary/pdf.css and reformatted to make legible

**Comment by kehh** [comment]
Report with multiple images and all fields populated
[print_summary.pdf](https://github.com/rwahs/providence/files/370785/print_summary.pdf)

Report with single image and fields missing data
[print_summary_single_image.pdf](https://github.com/rwahs/providence/files/370784/print_summary_single_image.pdf)

File: app/printTemplates/summary/local/ca_objects_summary.php [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  </span>
+<?php endif; ?>
+
+<?php
+	foreach($va_placements as $vn_placement_id => $va_bundle_info){
+		if (!is_array($va_bundle_info)) break;
+
+		if (!strlen($vs_display_value = $t_display->getDisplayValue($t_item, $vn_placement_id, array('purify' => true)))) {
+			if (!(bool)$t_display->getSetting('show_empty_values')) { continue; }
+			$vs_display_value = '';
```
Comment (kehh) [inline app/printTemplates/summary/local/ca_objects_summary.php:96]: This differs from upstream as they output `<not defined>`  here.

File: app/printTemplates/summary/local/summary.php [hunk 8]
```diff
@@ -0,0 +1,69 @@
+<?php
```
Comment (kehh) [inline app/printTemplates/summary/local/summary.php:1]: This is the default summary (not the object one). Same contents as `local/ca_objects_summary.php` but without the different image handling.

**Comment by mostro20** [comment]
CSS and print. Is there a more beautiful thing?

**Pull request merged**